frontend/ui/data/creoptions.lua

```diff
@@ -46,6 +46,8 @@ local CreOptions = {
                 default_arg = "page",
                 event = "SetViewMode",
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[- 'scroll' mode allows you to scroll the text like you would in a web browser (the 'Page Overlap' setting is only available in this mode).
+- 'page' mode splits the text into pages, at the most acceptable places (page numbers and the number of pages may change when you change fonts, margins, styles...).]]),
```

Comment:
I personally prefer an actual phrase (like etc., et cetera, and so forth), but otherwise it should be `, …`